### PR TITLE
ci-biology/rnaplex: Move to EAPI=5; Update homepage and SRC_URI.

### PR DESCRIPTION
**Pull request by marbre**
The URL to the src was outdated, therefore the manifest had to be updated.

### CODE REVIEW

File: sci-biology/rnaplex/rnaplex-0.2.ebuild [hunk 1]
```diff
@@ -16,9 +16,9 @@ IUSE=""
 KEYWORDS="~amd64 ~x86"
 
 DEPEND=""
-RDEPEND=""
+RDEPEND="${DEPEND}"
 S=${WORKDIR}/${MY_P}
 
 src_install() {
-	emake DESTDIR="${D}" install || die "install failed"
```
Comment (jlec) [inline sci-biology/rnaplex/rnaplex-0.2.ebuild]: You can drop the whole phase as this is the default implementation.

File: sci-biology/rnaplex/rnaplex-0.2.ebuild [hunk 2]
```diff
@@ -16,9 +16,9 @@ IUSE=""
 KEYWORDS="~amd64 ~x86"
 
 DEPEND=""
-RDEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (jlec) [inline sci-biology/rnaplex/rnaplex-0.2.ebuild]: if DEPEND and RDEPEND are empty, simply drop them.

File: sci-biology/rnaplex/rnaplex-0.2.ebuild [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-EAPI=2
+EAPI=5
 
 MY_P=RNAplex-${PV}
 
 DESCRIPTION="RNA-RNA interaction search"
-HOMEPAGE="http://www.tbi.univie.ac.at/~htafer/"
-SRC_URI="http://www.tbi.univie.ac.at/~htafer/RNAplex/${MY_P}.tar.gz"
+HOMEPAGE="http://www.bioinf.uni-leipzig.de/Software/RNAplex/"
+SRC_URI="http://www.bioinf.uni-leipzig.de/Software/RNAplex/RNAplex-0.2.tar.gz"
```
Comment (jlec) [inline sci-biology/rnaplex/rnaplex-0.2.ebuild]: you can still use MY_P here which makes future updates easier.

Comment (marbre) [inline sci-biology/rnaplex/rnaplex-0.2.ebuild]: Copy&Paste mistake while greping the new URI. Fixed.

**Comment by marbre** [comment]
The new commit should fix this. Thanks for the support :+1:

**Pull request merged**